src/DynamoCore/Search/SearchLibrary.cs

```diff
@@ -62,7 +62,7 @@ protected virtual void OnItemProduced(TItem item)
     /// <summary>
     ///     Utility methods for categorizing search elements.
     /// </summary>
-    public static class SearchCategory
+    public static class SearchCategoryUtil
     {
```

Comment:
This must be renamed as soon as LibraryUI introduces class with identical name `SearchCategory`.